0x.swift/Ethereum/EthereumLog.swift

```diff
@@ -0,0 +1,27 @@
+//
+// This source file is part of the 0x.swift open source project
+// Copyright 2019 The 0x.swift Authors
+// Licensed under Apache License v2.0
+//
+// EthereumLog.swift
+//
+// Created by Igor Shmakov on 11/05/2019
+//
+
+import Foundation
+import Web3Swift
+
+public struct EthereumLog {
```

Comment:
For library purposes, I guess we only need `address` to check that log belongs to a specific contract, `logIndex` to understand their order, `topics` and `data`